enterprise/signals.py

```diff
@@ -133,7 +141,7 @@ def default_content_filter(sender, instance, **kwargs):     # pylint: disable=un
 @receiver(post_save, sender=EnterpriseCustomerUser)
 def assign_or_delete_enterprise_admin_role(sender, instance, **kwargs):     # pylint: disable=unused-argument
     """
-    Assign or delete enterprise_admin role for EnterpriseCustomerUser when updated.
+    Assign or delete enterprise_admin role for EnterpriseCustomerUser when updated. Create third party analytics user.
```

Comment:
Please move `Create third party analytics user.` to next line.

Reply:
Done.